content/dstu2/authorization.md

```diff
@@ -93,45 +93,104 @@ The [bearer access token][7] returned from the authorization server is what you
 If the scopes "openid" and "profile" were originally provided, an [OpenID Connect][8] id_token will be included per the [SMART][4] specification that includes the user's URL (userfhirurl) as the "profile" claim. This is typically a link to a FHIR<sup>®</sup> standard Practitioner resource.
 
 ```
-{ 
+{
   "access_token" : "18adCadfj_lj13S3ada8.41jVCo_dgalL",
   "encounter" : "91731344",
   "patient" : "121341578",
   "id_token" : "eyAiYWxnIjoibm9uZSIKfQ==.ewogICJpc3MiOiAiaHR0cHM6Ly9hdXRob3JpemF0aW9uLmRldmNlcm5lci5jb20vb2F1dGgyIiwKICAic3ViIjogIm1yNTE0QGNlcm5lci5jb20iLAogICJhdWQiOiAiY2xpZW50X2FwcCIsCiAgIm5vbmNlIjogIm4tMFM2X1d6QTJNaiIsCiAgImV4cCI6IDEzMTEyODE5NzAsCiAgImlhdCI6IDEzMTEyODA5NzAsCiAgInByb2ZpbGUiOiJodHRwczovL2V4YW1wbGUuY29tLzEyMzQiCn0="
 }
 ```
 
-
 ### Using an access token ###
 In order to use an access token, your client application needs to provide the access token received from the authorization server to the protected resource. The following is a non-normative example of the usage of the token to access a protected RESTful web service on a resource server.  It will need to be added as an authorization HTTP header.
 
 ```
 Authorization: Bearer {ACCESS_TOKEN}
 ```
 
-If the access token is valid, and your application is authorized, the resource server will allow your client application to access its protected resources.
+If the access token is valid and your application is authorized, the FHIR resource server will allow your client application to access its protected resources.
```

Comment:
Do we need to say FHIR here? Why not stick to the general auth term? If we need FHIR, licensing requires it to have the reserved trademark and "FHIR standard" to indicate that this is an implementation of the FHIR standard rather than a brand of ours/something we own.